src/psiactionlist.cpp

```diff
@@ -530,7 +530,7 @@ void PsiActionList::Private::createChat()
 void PsiActionList::Private::createGroupchat()
 {
     {
-        IconAction *actInfo = new IconAction(tr("Information"), QLatin1String("psi/info"), tr("Information"), 0, this);
+        IconAction *actInfos = new IconAction(tr("Informations"), QLatin1String("psi/info"), tr("Informations"), 0, this);
```

Comment:
There is no such a word like "Informations"

Reply:
Here: https://i.imgur.com/2eTFChM.png

Several informations.

Reply:
There is no plural form of "information" word in English language.

Reply:
On GitHub:
- https://github.com/search?o=desc&q=informations&s=indexed&type=Code
- https://github.com/search?q=informations&type=code

Reply:
I don't care about mistakes of other people.